templates/bundle_version.sh

```diff
@@ -0,0 +1,16 @@
+GIT=$(sh /etc/profile; which git)
+GIT_RELEASE_VERSION=$(${GIT} describe --tags --always --abbrev=0)
+COMMITS=$(${GIT} rev-list HEAD | wc -l)
+COMMITS=$(($COMMITS))
```

Comment:
Aha:

```shell
$ commits="$(git rev-list head | wc -l)"
$ echo $commits
     257
$ foo=$((commits))
$ echo $foo
257
```

Looks like it's to remove the leading spaces from `wc -l`. So is there a better way to do this?